docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Day 2 Operations with Keptn
+description: How to operate and maintain your Keptn Apps
+weight: 500
+---
+
+After you have successfully rolled out your application by following
+the instructions in the [integration guide](../integrate),
+Keptn also assists you with day 2 operations for your application.
+
+Tasks that fall under this category include:
+
+* Updating the version of one or more workloads that are part of
+the same application.
+* Adding a new workload to an existing application
+* Monitoring the health of your application using `KeptnMetrics`
+* Optimizing the resource usage of your applications by integrating
+`KeptnMetrics` into a
+[HorizontalPodAutoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+## Updating Workload Versions
+
+After a first successful deployment of your application with Keptn,
+you will eventually need to update the version of one or
+more workloads that are part of the application.
+This usually involves updating the image of a deployment
+and changing the configuration of a deployment.
+For example, using a different service account name for a pod
+could be seen as an update.
+Regardless of that, however, it is the user who decides what Keptn
+sees as a version bump in the application by setting the value of
+the `keptn.sh/version` or `app.kubernetes.io/version` labels/annotations
+in their workloads.
+
+When this changes, Keptn interprets a change as a new version
+and thus re-runs the pre- and post-tasks and evaluations for the application.
+
+If the version label/annotation does not change, Keptn does not consider
+a change of a workload configuration to be an update, and therefore no pre-
+and post-tasks/evaluations are executed because they have already been
+completed for the related `WorkloadVersion`.
+
+To illustrate the update of a workload,
+let's assume the following example, including
+a workload called `podtato-head-frontend` that includes a pre-task and
+a pre-evaluation.
+
+```yaml
```

Comment:
please use external asset files and include them here for all the code examples in this file :)